pkg/cluster/cluster.go

```diff
@@ -200,6 +200,45 @@ func (c *Cluster) initUsers() error {
 	return nil
 }
 
+func CreateSecrets() []Action {
+	return nil
+}
+
+func (c *Cluster) PlanForCreate() (plan []Action) {
+	plan = append(plan, c.PlanForSecrets()...)
+	return plan
+}
+
+// TODO: mind the secrets of the deleted/new users
+func (c *Cluster) PlanForSecrets() (plan []Action) {
```

Comment:
it is worth commenting here what a plan actually means (sequence of actions i assume? )  It is not immediately obvious for a person w/o operator and DB background